edx_event_bus_kafka/internal/consumer.py

```diff
@@ -132,6 +133,24 @@ def _shut_down(self):
         """
         self._shut_down_loop = True
 
+    def _db_needs_reconnect(self):
+        """
+        Determine if we need to try to reconnect to the database
+
+        Returns:
+            True if we have a db connection but it is unusable
+            False otherwise (if we don't have a connection or our current connection is fine)
+        """
+        return connection.connection and not connection.is_usable()
+
+    def _reconnect_to_db(self):
+        """
+        Pass-through to connection.connect()
```

Comment:
What about something like:
> Used to re-connect broken connections by calling connection.connect()

Note: even if we change our mocking (see other comment), you may still want this wrapper to have a better name and docstring. But, maybe it is overkill. Not sure.

Reply:
I like calling out that it's a straight up passthrough, but maybe I'll put that in the second line

Reply:
[fixed] although I kept my line as the more detailed explanation